lib/kredis/version.rb

```diff
@@ -1,3 +1,3 @@
 module Kredis
-  VERSION = "0.4.0"
+  VERSION = "0.4.1"
```

Comment:
We'll change the version when we prepare a release, we shouldn't do this in this PR.